site/content/entry/yuzu-progress-report-mar-2020/index.md

```diff
@@ -0,0 +1,131 @@
++++
+date = "2020-04-19T04:00:00-03:00"
+title = "Progress Report March 2020"
+author = "GoldenX86"
+forum = 217502
++++ 
+
+How are you doing in this quarantine, yuz-ers? We hope you are all staying in your homes safe and sound.
```

Comment:
`yuz-ers`
sounds way too close to losers.




:P

Reply:
Thanks, now I can't unsee it. This wasn't me, Vortex likes it, looks like it was used before.